docs/authorization-configuration.md

```diff
@@ -36,7 +36,9 @@ auth: {
 - **scope** defines the permissions to request at login.
 - **redirect_uri** sets the URL to return to after login. The default application root is `/`.
 - **post_logout_redirect_uri** sets the URL to return after logout. The default URL is `/logout.html`.
-- **automaticSilentRenew** enables the renewal of the automatic token if it is supported by the server. The default value is `false`.
+- **automaticSilentRenew** enables the automatic silent renewal of the token if it is supported by the server. The default value is `false`. For this mechanism to work, browser must have third-party cookies support enabled.
+- **accessTokenExpiringNotificationTime** is the number of seconds before an access token is to expire and triggers silent token refresh. The default value is 60.
+- **thirdPartyCookiesScriptLocation** is the URL to the page containing third-party cookies support check. See more about it in [Third-party cookies and silent token refresh section](#Third-party-cookies-and-silent-token-refresh). 
```

Comment:
`For details, see [Third-party cookies and silent token refresh section](#Third-party-cookies-and-silent-token-refresh). `

Reply:
fixed